CSharpOOP4/Ball.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Kristina_Kulich__Application.CSharpOOP4;
+
+public class Ball : Shape
+{
+    public string _name;
```

Comment:
1. fields should be private.
2. Add space between fields.
3. Remove one space between fields and constructor.

Reply:
fixed